presto-docs/src/main/sphinx/release/release-0.223.rst

```diff
@@ -0,0 +1,26 @@
+=============
+Release 0.223
+=============
+
+General Changes
+---------------
+
+* Fix a bug where a connector may not optimize query plan even when an optimizer is specified.
+* Support multibyte characters in ``date_format`` function.
+* Add support for using long polling for task info updates to reduce cpu load on the coordinator.
+  Set experimental.task.info-update-refresh-max-wait to enable the feature.
```

Comment:
Specify if this is a session or configuration property. In addition to that the name of the prop should be backticked.

```
This can be enabled by setting the ``experimental.task.info-update-refresh-max-wait`` configuration property.
```